project.yaml

```diff
@@ -907,14 +907,47 @@ Layer:
               ('railway_' || (CASE WHEN railway IN ('platform') THEN railway ELSE NULL END)),
               (('aeroway_' || CASE WHEN aeroway IN ('runway', 'taxiway', 'helipad') THEN aeroway ELSE NULL END))
             ) AS feature
-          FROM planet_osm_polygon
-          WHERE highway IN ('residential', 'unclassified', 'pedestrian', 'service', 'footway', 'living_street', 'track', 'path', 'platform', 'services')
-            OR railway IN ('platform')
-            OR aeroway IN ('runway', 'taxiway', 'helipad')
+          FROM (
+          SELECT 
+              way, highway, railway, aeroway, z_order, way_area FROM planet_osm_polygon
+            WHERE highway IN ('residential', 'unclassified', 'pedestrian', 'service', 'footway', 'living_street', 'track', 'path', 'platform', 'services')
+              OR railway IN ('platform')
+              OR aeroway IN ('taxiway', 'helipad')
+          UNION ALL
+          SELECT -- runways with width either tagged or estimated from runway length
+              ST_Buffer(
+                way,
+                -- more generic estimation for scale factor on arbitrary projections would be:
+                -- 0.5*(St_Distance(ST_StartPoint(way), ST_EndPoint(way))/St_Distance_Sphere(ST_Transform(ST_StartPoint(way), 4326), ST_Transform(ST_EndPoint(way), 4326)))*
```

Comment:
This still wouldn't work on arbitrary projections. The most common other projection is 4326, and st_buffer appear to work differently in different directions when the result is rendered in mercator